backend/infrahub/graphql/initialization.py

```diff
@@ -49,6 +49,11 @@ def active_account_session(self) -> AccountSession:
             return self.account_session
         raise InitializationError("GraphQLContext doesn't contain an account_session")
 
+    def get_service(self) -> InfrahubServices:
+        if not self.service:
+            raise ValueError("GraphqlContext doesn't have a valid service object")
+        return self.service
```

Comment:
I think we should stick to the property approach that we have for the account_session just above, mostly to avoid mixing properties and methods for what is really the same type of thing.